### PR TITLE
speed up building by only building for trusty

### PR DESCRIPTION
**Pull request by n1koo**
Only build for trusty

@Shopify/borg @csfrancis @shivnagarajan

### CODE REVIEW

File: hack/make/build-deb [hunk 1]
```diff
@@ -38,7 +38,7 @@ set -e
 	# TODO decide if it's worth getting go-md2man in _each_ builder environment to avoid this
 
 	# TODO add a configurable knob for _which_ debs to build so we don't have to modify the file or build all of them every time we need to test
```
Comment (shivnagarajan) [inline hack/make/build-deb:40]: can we not just fix the TODO and submit a patch upstream?

Comment (n1koo) [inline hack/make/build-deb:40]: Sure. Not sure what kind of knob they are after, but having an ENV supplied to `make` would be trivial`

Comment (shivnagarajan) [inline hack/make/build-deb:40]: i was thinking something as simple as that, with the default in the absence of the env var being `*`

Comment (n1koo) [inline hack/make/build-deb:40]: Yup. I can do that :+1:

**Pull request merged**